### DISCUSSION TITLE
Ravagers afraid of rabbits

### DISCUSSION THREAD

**Discussion by MrRaposinha** [PurpurMC/Purpur#713] [__Completed__] [answered]
I found this old twitter where it says that ravagers were originally afraid of rabbits, I think it would be nice to implement this again through a config option, maybe?

(Sorry if something is wrong, my english is not the best.)

![Jeb_](https://user-images.githubusercontent.com/43957227/137819944-a4a15790-e65f-4c7f-88b5-6cc466e8c327.png)

**Comment by granny** [comment, marked as answer]
Implemented as of build `#2188` https://github.com/PurpurMC/Purpur/commit/a398e71e9d559597b85108ca91b929738352582e. It defaults to `false`.
